### PR TITLE
Run isort with the pre-commit tool

### PR DESCRIPTION
**Pull request by MVrachev**
In the .pre-commit-config.yaml file when we call isort we pass the current directory as a location where isort should lint. This in turn overrides all excludes made by the pre-commit tool as after that, we call isort upon all folders.

For example, I have a local virtual environment with the name ".env" and even if I add the "exclude" option for isort it doesn't work. Now, when we don't provide the target folders to isort the pre-commit tool does that and successfully ignores the ".env" folder.

As a reference see this discussion: https://github.com/pre-commit/mirrors-mypy/issues/1#issuecomment-472968707

Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

### CODE REVIEW

**Comment by MVrachev** [comment]
I will love to hear @KAUTH opinion on this as he is more familiar with the `pre-commit` tool.

**Comment by KAUTH** [comment]
@MVrachev very good catch, I believe your analysis is correct. As explained in the link you shared, `exclude` only acts in the files initially passed to by the `pre-commit` tool (something like `git ls-files --`). We should correct this in the other repos as well.
+1 from me

**Review by kairoaraujo — APPROVED** [review]

**Comment by MVrachev** [comment]
Do we want to wait for another approval @kairoaraujo or should I directly propose such prs in the other repositories?

**Comment by kairoaraujo** [comment]
No. You can merge it.

**Pull request merged**
